src/components/Projects.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import styled from 'styled-components';
+import data from '../data.json';
+import { OuterWrapper, InnerWrapper, SectionTitle, Tag } from './reusable/Wrappers';
+
+export const Projects = () => {
+  const ProjectSection = styled.div`
+    padding: 75px 0px 10px 0px;
+    background-color: white;
+  `;
+
+  const ProjectWrapper = styled.div`
+    display: flex;
+    justify-content: center;
+    margin-top: 60px;
+  `;
+
+  const CardWrapper = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-content: space-between;
+    width: 750px;
+    height: fit-content;
+    margin: 10px;
+  `;
+
+  const CardContent = styled.div`
+    width: 335px;
+    margin: 10px;  
+      img {
+        width: 325px;
+        height: 325px;
+        display: block;
+      }
+  `;
+
+  const ProjectBackground = styled.img`
+    position: relative;
+    display: flex;
+    width: 325px;
+    height: 325px; 
+    margin: 0 auto;
+    &:hover{
+      display: none;
+    }
+  `;
+
+  const CardOverlay = styled.div`
+    position: relative;
```

Comment:
My code looks like your but with this changes for the overlay.
position: absolute;